scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$IMGVERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
+ACI_GOPATH=/go
+VERSION=${VERSION:-"v0.10.0"}
+# if [ `expr match $VERSION '.*git$'` -gt 0 ]; then
+#     VERSION="$VERSION$(git rev-parse --short HEAD 2>/dev/null)$(git diff-index --quiet HEAD 2>/dev/null || echo -dirty)"
+# fi
+echo "Version: $VERSION"
+
+echo "Building $ACIFILE"
+
+which docker2aci || go get github.com/appc/docker2aci
```

Comment:
Oh yeah, and when setting GOPATH, probably PATH will need to be adapted to point to bin dir in GOPATH.